test/sampling_correlation_matrices_test.cpp

```diff
@@ -64,6 +79,33 @@ void check_output(PointList &randPoints, int num_points, int n){
     CHECK(score.maxCoeff() < 1.1);
 }
 
+template<typename NT, typename VT, typename MT>
+void check_output_MT(std::list<MT> &randCorMatrices, int num_points, int n){
```

Comment:
plz fix indentation in this function

Reply:
really sorry for overlooking this. i fixed it